src/main/java/org/jenkinsci/bytecode/Transformer.java

```diff
@@ -57,16 +58,28 @@ public synchronized void loadRules(Collection<? extends ClassLoader> loaders) th
             return image;
 
         final ClassReader cr = new ClassReader(image);
-        final ClassWriter cw = new ClassWriter(/*ClassWriter.COMPUTE_FRAMES|*/ClassWriter.COMPUTE_MAXS);
+        final ClassWriter cw = new ClassWriter(ClassWriter.COMPUTE_FRAMES|ClassWriter.COMPUTE_MAXS);
```

Comment:
I saw this line of code a few days ago and it made me wonder when and why was `ClassWriter.COMPUTE_FRAMES` removed. Maybe the file history tells when that was changed and if there was a JIRA etc explaining why.